scheduler/src/cook/regexp_tools.clj

```diff
@@ -0,0 +1,32 @@
+;;
+;; Copyright (c) Two Sigma Open Source, LLC
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;  http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+(ns cook.regexp-tools
```

Comment:
Refactored here. 

Got a crazy serialization error making the uberjar when I made cook.rate-limit depend on cook.tools (and its dependencies), so moved this here.

Reply:
Does this still need to move?

Reply:
No, but I don't think it harms anything to move it, do you want it to move back?

Reply:
I'm fine leaving it like this if we think it's better